src/cli_agent_orchestrator/constants.py

```
@@ -63,7 +63,9 @@ def _env_int(name: str, default: int) -> int:
 TERMINAL_LOG_DIR.mkdir(parents=True, exist_ok=True)
 
 # FIFO directory for event-driven terminal output streaming
-FIFO_DIR = CAO_HOME_DIR / "fifos"  # Named pipes for tmux pipe-pane streaming
+# Use /tmp instead of CAO_HOME_DIR to avoid WSL2 Windows mount limitations
+# (WSL2 doesn't support FIFO pipes on /mnt/c filesystem)
+FIFO_DIR = Path("/tmp") / "cli-agent-orchestrator" / "fifos"  # Named pipes for tmux pipe-pane streaming
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)